.github/workflows/auto-update-collector-version.yml

```diff
@@ -1,27 +0,0 @@
-name: Check collector version
-
-on:
-  workflow_dispatch:
-  schedule:
-    - cron: '0 1 * * *'
-
-jobs:
-  build:
-    # Make sure that the workflow does not run on forks (add yourself as exception if you work on the workflow)
-    if: github.repository_owner == 'open-telemetry' || github.repository_owner == 'svrnm'
-    name: Check collector version
-    runs-on: ubuntu-20.04
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-
-      - name: Auto-update
-        run: ./.github/workflows/scripts/check-collector-version.sh
-        env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          GIT_AUTHOR_NAME: opentelemetrybot
-          GIT_COMMITTER_NAME: opentelemetrybot
-
-          # Set opentelemetrybot's email, otherwise CLA checks will fail
-          GIT_AUTHOR_EMAIL: 107717825+opentelemetrybot@users.noreply.github.com
-          GIT_COMMITTER_EMAIL: 107717825+opentelemetrybot@users.noreply.github.com
```

Comment:
Do any of these env vars need to be set in the new workflow too?

Reply:
no, these are handled by

```
          git config user.name opentelemetrybot
          git config user.email 107717825+opentelemetrybot@users.noreply.github.com
```